client/app/bundles/course/assessment/pages/AssessmentStatistics/StudentMarksPerQuestionTable.tsx

```diff
@@ -143,6 +143,26 @@ const StudentMarksPerQuestionTable: FC<Props> = (props) => {
       ),
       csvDownloadable: true,
     },
+    {
+      searchProps: {
+        getValue: (datum) => datum.courseUser.email,
+      },
+      title: 'Email',
+      sortable: true,
+      searchable: true,
+      visible: false,
+      cell: (datum) => (
+        <div className="flex grow items-center">
+          <Link to={`/courses/${courseId}/users/${datum.courseUser.id}`}>
+            {datum.courseUser.email}
+          </Link>
+          {datum.courseUser.isPhantom && (
+            <GhostIcon className="ml-2" fontSize="small" />
+          )}
+        </div>
```

Comment:
Just include the email as text is fine, no need to display the GhostIcon twice, no need to link to user twice.